instrumentation/opentelemetry-instrumentation-botocore/src/opentelemetry/instrumentation/botocore/extensions/dynamodb.py

```diff
@@ -350,12 +380,25 @@ class _DynamoDbExtension(_AwsSdkExtension):
     def __init__(self, call_context: _AwsSdkCallContext):
         super().__init__(call_context)
         self._op = _OPERATION_MAPPING.get(call_context.operation)
+        self._configuration = (
+            call_context.configuration.get_dynamodb_configuration()
+        )
 
     def extract_attributes(self, attributes: _AttributeMapT):
         attributes[SpanAttributes.DB_SYSTEM] = DbSystemValues.DYNAMODB.value
         attributes[SpanAttributes.DB_OPERATION] = self._call_context.operation
         attributes[SpanAttributes.NET_PEER_NAME] = self._get_peer_name()
 
+        if self._call_context.operation in _db_statement_operations:
+            attributes[SpanAttributes.DB_STATEMENT] = str(
+                self._call_context.params
+            )
+
+            if self._configuration.get("sanitize_query"):
+                attributes[
+                    SpanAttributes.DB_STATEMENT
+                ] = _conv_params_to_sanitized_str(self._call_context.params)
```

Comment:
I get what you're saying. However, there are several similar merged PRs (#1545, #1547, #1548), and an open discussion in [specs](https://github.com/open-telemetry/opentelemetry-specification/issues/3150). Your call :)

Reply:
Does it then even make sens to add statement sanitization if the format isn't specced yet? Also when considering that there doesn't seem to be a common query language for DynamoDB (e.g SDK input parameters vs PartiQL).

Personally i don't have a strong opinion about statement capturing/sanitization and i'm just pointing out that it might lead to breaking changes in the future. 
If you think this is ok feel free and continue with merging this PR :)